packages/terra-worklist-data-grid/src/subcomponents/Cell.jsx

```diff
@@ -142,7 +142,7 @@ function Cell(props) {
   const onMouseDown = ((event) => {
     if (!isFocusTrapEnabled) {
       onCellSelect({
-        rowId, columnId, rowIndex, columnIndex, multiSelect: event.shiftKey, selectedByKeyboard: false, isCellSelectable: (!isMasked && isSelectable),
+        rowId, columnId, rowIndex, columnIndex, isShiftKeyPressed: event.shiftKey, isCellSelectable: (!isMasked && isSelectable),
```

Comment:
Should we simplify the name to isShiftPressed?

Reply:
Done